Automattic/sensei
# Add editor player hook

Fixes 1505-gh-Automattic/sensei-pro

### Changes proposed in this Pull Request

* It contains some improvements suggested in [this code review](https://github.com/Automattic/sensei/pull/5364).
* It includes a new React hook to allow the use of the player helper on the editor side.

### Testing instructions


* In Sensei Pro checkout `feature/interactive-video-block`.
* Extract the file [temp.diff.zip](https://github.com/Automattic/sensei/files/9221177/temp.diff.zip). In Sensei Pro, apply this diff with `git apply temp.diff`.
* Add some interactive video blocks to the editor of all the types (Video, VideoPress, YouTube, and Vimeo).
* Make sure you see the duration and the current time of the video while playing it.
* Try to update the videos to another one, and make sure it updates the video duration.
* And also try to make weird things in order to catch edge cases. 🤪


### Screenshot / Video

https://user-images.githubusercontent.com/876340/181075737-8550f1da-26eb-4087-8adc-d0d5a350ffdc.mov
